warehouse/filters.py

```diff
@@ -136,3 +137,8 @@ def format_classifiers(classifiers):
     structured = collections.OrderedDict(sorted(structured.items()))
 
     return structured
+
+
+def filter_valid_uris(items):
+    """ Filter out a list so that it only contains valid urls """
```

Comment:
I don't think docstrings tend to start or end with whitespace. They should end with a period.